specs/phase0/validator.md

```diff
@@ -604,3 +605,11 @@ Specifically, when signing an `Attestation`, a validator should perform the foll
 2. Generate and broadcast attestation.
 
 If the software crashes at some point within this routine, then when the validator comes back online, the hard disk has the record of the *potentially* signed/broadcast attestation and can effectively avoid slashing.
+
+## Protection best practices
+A validator client should be considered standalone and should consider the node it's connected to as un-trusted. This means that the validator client should protect: 
+1) Private keys - private keys should be protected from ever being exported out accidentally or by an attacker. Attestations and blocks should be signed internally in the process itself, keys should always be encrypted on disk.
```

Comment:
```suggestion
1) Private keys -- private keys should be protected from being exported accidentally or by an attacker.
```
Second sentence is not mandatory. It makes sense to keep a key in an HSM, for example. Since processes are not protected entities of any kind, it is not a security improvement to require signing in the same process. Also, encrypting keys on disk does not add security if the password is also stored on disk, which is a necessity for validator nodes to be able to restart automatically.

Reply:
You could hold the password in an HSM which is more likely to happen sooner as it will take time for BLS keys to be supported. 
I see it as best design practice for future improvements, which is better than just leaving keys as plain text.
But you are right that for now it's probably very similar inn terms of security.

Reply:
Of course an HSM makes sense. But the security of
1. A validator client running form unencrypted keys
2. A validator client running with encrypted keys, with the password given on the command line (so stored on disk to load when the computer starts up)

is the same. Case 2 might feel safer but is security theatre.